src/pages/guide/css/index.md

```diff
@@ -41,7 +41,8 @@ Make sure to carefully read and follow the instructions in the tutorials below.
 * [MDN's CSS reference](https://developer.mozilla.org/en-US/docs/Web/CSS)
 * [w3school's CSS reference](https://www.w3schools.com/cssref/default.asp)
 * [w3school's CSS reference](https://www.w3schools.com/cssref/default.asp)
-* [Game to master Flexbox](https://mastery.games/p/flexbox-zombies)
+* [Flexbox Game](https://mastery.games/p/flexbox-zombies)
```

Comment:
For this resource, can you rename it to be `Flexbox Zombies Game` to match the title on the game itself?